web/src/popover_menus.js

```diff
@@ -188,6 +188,23 @@ function on_show_prep(instance) {
     popovers.hide_all_except_sidebars();
 }
 
+// This function is created to instantly show the popover menu instead of
+// registering events on `body`. This is useful for cases when we,
+// for example, trigger the popover by a keyboard shortcut.
+export function show_popover_menu(target, popover_props) {
+    const instance = target._tippy;
+
+    if (instance) {
+        instance.hide();
+        return;
+    }
+
+    tippy(target, {
+        ...default_popover_props,
+        showOnCreate: true,
+        ...popover_props,
+    });
+}
```

Comment:
Should `register_popover_menu` be using this as a subroutine? It seems to mostly duplicate code from there.

Also in this commit you're missing a blank line after this function.

Reply:
Thanks! I added a follow-up commit with some refactoring.